pkg/result/processors/sort_results.go

```diff
@@ -31,6 +31,10 @@ var _ Processor = (*SortResults)(nil)
 
 type issueComparator func(a, b *result.Issue) int
 
+// SortResults sorts reports based on criteria:
+// - file names, line numbers, positions
+// - linter names
+// - severity names
```

Comment:
Perhaps we should add a newline before the first list item:

<img width="893" alt="image" src="https://github.com/user-attachments/assets/c8c5cb88-b754-4e64-9206-7e6f343d1f58" />